### DISCUSSION TITLE
How to add link in label

### DISCUSSION THREAD

**Discussion by chesterlaykin** [formkit/formkit#782] [General]
I am wondering how to add a link inside a label.
For instance,here you have a checkbox to accept terms [checkbox](https://formkit.com/essentials/validation#accepted)
but it would be nice to have a link to the terms?

**Comment by luan-nk-nguyen** [comment]
Hi @chesterlaykin — You can do it with slots or the sections-schema feature. Example here: https://formkit.link/577824c5b5d563120e20abcaecc0c051

For future reference, this example is on our examples page! https://formkit.com/essentials/examples

Cheers.

**Reply by chesterlaykin** [reply]
Thanks for the reply, sounds great, I'll try it!
